### PR TITLE
PWN-5330 fix onboarding text and some bugs

### PR DESCRIPTION
**Pull request by davran1001**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-5330

### CODE REVIEW

File: app/release/output-metadata.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (gslevinkov) [inline app/release/output-metadata.json:1]: ????

File: app/src/main/java/org/p2p/wallet/auth/repository/UserSignUpDetailsStorage.kt [hunk 2]
```diff
@@ -24,12 +24,14 @@ class UserSignUpDetailsStorage(
...
     )
 
-    fun save(data: Web3AuthSignUpResponse, userId: String): Boolean {
+    fun save(data: Web3AuthSignUpResponse, userId: String, isCreate: Boolean = true): Boolean {
         val key = generatePrefsKey(userId)
         val value = SignUpUserDetails(userId, data)
 
         accountStorage.saveObject(KEY_LAST_DEVICE_SHARE_ID, value)
-        accountStorage.saveString(KEY_IN_SIGN_UP_PROCESS, key)
+        if (isCreate) {
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/auth/repository/UserSignUpDetailsStorage.kt:32]: isCreateFlow?
what about removing this key?

File: app/src/main/res/values/onboarding_strings.xml [hunk 3]
```diff
@@ -129,4 +129,8 @@
     <string name="onboarding_offline_error">☕ You\'re offline. Keep calm</string>
     <string name="onboarding_google_services_error">❌ There is a problem with Google Services. Try again</string>
     <string name="onboarding_with_email">%s</string>
+    <string name="onboarding_if_you_want_continue_with_email">If you want continue with %s</string>
+    <string name="restore_select_phone_number_if_you_made_a_mistake"> select phone number. If you made a mistake, please choose another mail</string>
+    <string name="restore_use_your_social_account">Use your social account to continue</string>
+    <string name="onborading_create_a_new_wallet">Create a new wallet</string>
```
Comment (gslevinkov) [inline app/src/main/res/values/onboarding_strings.xml:135]: onborading -> onboarding

**Review by gslevinkov — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/p2p/wallet/auth/repository/UserSignUpDetailsStorage.kt (hunk 2 above)
Comment (davran1001) [reply app/src/main/java/org/p2p/wallet/auth/repository/UserSignUpDetailsStorage.kt:32]: this key remove only on finish create flow,  and remove after create wallet is finish

File: app/release/output-metadata.json (hunk 1 above)
Comment (davran1001) [reply app/release/output-metadata.json:1]: removed this file

**Review by 1sakov — APPROVED** [review]

**Review by gslevinkov — APPROVED** [review]

**Review by kabramovp2p — APPROVED** [review]

**Pull request merged**